### PR TITLE
add faq.md

### PR DESCRIPTION
**Pull request by Xtrah**
- Add a FAQ file for frequently asked questions

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# FAQ
+
+Spørsmål og svar kommer her.
```
Comment (halvose) [inline faq.md:3]: Unnecessary space here

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# FAQ
+
```
Comment (AdrianFolge) [inline faq.md:2]: Kommentar

Comment (amunium) [inline faq.md:2]: Dette er en fantastisk kommentar

File: faq.md (hunk 1 above)
Comment (Xtrah) [reply faq.md:3]: No, according to markdown conventions, you need at least one space between lines.

**Comment by halvose** [comment]
LGTM!

**Pull request merged**